spec/infinispan_local_spec.js

```diff
@@ -194,6 +307,10 @@ describe('Infinispan local client', function() {
       tests.iterateEntries('local', 3, client)
     );
 
+    it('iterate over entries when batch size is 0',
+      tests.iterateEntries('local', 0, client)
```

Comment:
What is this expected to do? What does 0 value mean? The jsdoc does not explain. Did you investigate this to see what is supposed to do?

Reply:
Ah I see it below now. Please add info to the jsdoc.